apps/console/src/features/users/components/user-profile.tsx

```diff
@@ -244,8 +247,8 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
     /**
      * This will add role attribute to countries search input to prevent autofill suggestions.
      */
-    const onCountryRefChange = useCallback(node => {
-        if (node !== null) { 
+    const onCountryRefChange: any = useCallback((node: any) => {
```

Comment:
Do not use `any` since that's an antipattern for places where we can actually define a type.

If you are unsure of the type. Do the following.

1. Type the variable with an incorrect type.

```number
const onCountryRefChange: number
```

2. Then the compiler will complain and suggest you a type.

<img width="1288" alt="Screenshot 2022-11-21 at 10 22 32" src="https://user-images.githubusercontent.com/25959096/202968308-8c693a59-2f64-48ee-b4bb-7e36753f520f.png">

3. You can then define a type based on the suggestion.

Since `any` is not an option, we can check and find the actual type of the `element`.

`(element: HTMLElement) => void` is the correct type here.


```suggestion
    const onCountryRefChange: (element: HTMLElement) => void = useCallback((node: any) => {
```